src/i18n/nl.js

```diff
@@ -16,65 +16,65 @@ export default {
-      invalid_captcha: 'Los de uitdagingsvraag op om te verifiëren dat je geen robot bent.',
-      invalid_recaptcha: 'Selecteer het selectievakje om te verifiëren dat u geen robot bent.'
+      invalid_captcha: 'Los de vraag op om te verifiëren dat u geen robot bent.',
+      invalid_recaptcha: 'Selecteer het vakje om te verifiëren dat u geen robot bent.'
     },
     passwordless: {
       'bad.email': 'Het e-mailadres is ongeldig',
       'bad.phone_number': 'Het telefoonnummer is ongeldig',
       'lock.fallback': 'Onze excuses, er is iets fout gegaan.'
     },
     signUp: {
-      invalid_password: 'Wachtwoord is ongeldig.',
-      'lock.fallback': 'Onze excuses, er is iets mis gegaan bij de poging om te registreren.',
-      password_dictionary_error: 'Wachtwoord is een te bekend woord.',
-      password_no_user_info_error: 'Wachtwoord is gebaseerd op gebruikers informatie.',
-      password_strength_error: 'Wachtwoord is niet sterk genoeg.',
-      user_exists: 'Gebruiker bestaat al.',
-      username_exists: 'Gebruikersnaam bestaat al.',
+      invalid_password: 'Het wachtwoord is ongeldig.',
+      'lock.fallback': 'Onze excuses, er is iets mis gegaan bij het registreren.',
+      password_dictionary_error: 'Het wachtwoord is een te bekend woord.',
+      password_no_user_info_error: 'Het wachtwoord is gebaseerd op gebruikers informatie.',
+      password_strength_error: 'Het wachtwoord is niet sterk genoeg.',
+      user_exists: 'De gebruiker bestaat al.',
+      username_exists: 'De gebruikersnaam bestaat al.',
       social_signup_needs_terms_acception:
-        'Ga akkoord met de onderstaande Servicevoorwaarden om door te gaan.'
+        'Ga akkoord met de onderstaande servicevoorwaarden om door te gaan.'
     }
   },
   success: {
     logIn: 'Dank u voor het inloggen.',
-    forgotPassword: 'Wij hebben een e-mail verstuurd voor het resetten van uw wachtwoord.',
-    magicLink: 'Wij hebben een link verstuurd, naar %s, om mee in te loggen.',
+    forgotPassword: 'Wij hebben u een e-mail gestuurd voor het resetten van uw wachtwoord.',
+    magicLink: 'Wij hebben een link gestuurd, naar %s, om mee in te loggen.',
     signUp: 'Bedankt voor uw aanmelding.'
   },
   blankErrorHint: '',
   blankPasswordErrorHint: 'Mag niet leeg zijn',
   blankEmailErrorHint: 'Mag niet leeg zijn',
   blankUsernameErrorHint: 'Mag niet leeg zijn',
   blankCaptchaErrorHint: 'Mag niet leeg zijn',
-  codeInputPlaceholder: 'uw code',
+  codeInputPlaceholder: 'Uw code',
   databaseEnterpriseLoginInstructions: '',
   databaseEnterpriseAlternativeLoginInstructions: 'of',
   databaseSignUpInstructions: '',
   databaseAlternativeSignUpInstructions: 'of',
-  emailInputPlaceholder: 'iets@example.com',
-  enterpriseLoginIntructions: 'Login met uw bedrijfs inloggegevens.',
-  enterpriseActiveLoginInstructions: 'Voer uw bedrijfs inloggegevens in op %s.',
+  emailInputPlaceholder: 'iemand@voorbeeld.nl',
```

Comment:
Not blocking, but I don't think this change adds a lot of value. Even more so it changes it to use a `.nl` domain, while dutch is spoken outside of the Netherlands. 

Personally, I would keep the `example.com` domain.